### PR TITLE
issue-55

### PR DESCRIPTION
**Pull request by brianorwhatever**
This adds the basic message didcomm protocol once a message is created it will send it over. On the other side when a message is received it creates on as well

### CODE REVIEW

File: backend/src/db/db.service.ts [hunk 1]
```diff
@@ -13,16 +13,18 @@ export class DBService {
...
-      auth: {
-        user: this.config.get('DBUSER'),
-        password: this.config.get('DBPASSWORD'),
-      },
+    let connectOptions = {
       reconnectTries: 15,
       reconnectInterval: 1000,
       connectTimeoutMS: 60000,
     };
+    if (this.config.get('DBAUTHSOURCE')) {
```
Comment (Vanderscycle) [inline backend/src/db/db.service.ts:21]: We need DBAUTHSOURCE in the env file?

File: backend/src/didcomm/handlers/basic-message.handler.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { IDIDComm } from '@aviarytech/didcomm-messaging';
+import {
+  BasicMessageMessageHandler,
```
Comment (Vanderscycle) [inline backend/src/didcomm/handlers/basic-message.handler.ts:3]: The var naming gets a bit confusing

File: backend/src/db/db.service.ts (hunk 1 above)
Comment (brianorwhatever) [reply backend/src/db/db.service.ts:21]: no not locally just on the server. make sure you don't have it locally or else you will trip this which we don't want

**Review by Vanderscycle — APPROVED** [review]

File: backend/src/didcomm/handlers/basic-message.handler.ts (hunk 2 above)
Comment (brianorwhatever) [reply backend/src/didcomm/handlers/basic-message.handler.ts:3]: agreed

**Pull request merged**